### PR TITLE
feat(browser): add browser iframe mouse interaction

### PR DESCRIPTION
**Pull request by userquin**
### Description

supersedes #5813

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. If the feature is substantial or introduces breaking changes without a discussion, PR might be closed.
- [ ] Ideally, include a test that fails without this PR but passes with it.
- [ ] Please, don't make changes to `pnpm-lock.yaml` unless you introduce a new test example.

### Tests
- [ ] Run the tests with `pnpm test:ci`.

### Documentation
- [ ] If you introduce new functionality, document it. You can run documentation with `pnpm run docs` command.

### Changesets
- [ ] Changes in changelog are generated from PR name. Please, make sure that it explains your changes in an understandable manner. Please, prefix changeset messages with `feat:`, `fix:`, `perf:`, `docs:`, or `chore:`.

### CODE REVIEW

File: packages/ui/client/components/BrowserIframe.vue [hunk 1]
```diff
@@ -84,10 +100,22 @@ async function changeViewport(name: string) {
...
+    <div flex-auto class="scrolls">
+      <div id="tester-ui" ref="testerRef" style="overflow: auto; width: 100%; height: 100%">
         Select a test to run
       </div>
     </div>
   </div>
 </template>
+
+<style>
+#tester-ui {
```
Comment (userquin) [inline packages/ui/client/components/BrowserIframe.vue:112]: use `#tester-ui:not(.testing)` and remove `#tester-ui.testing`  style

File: packages/ui/client/composables/browser.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  const panel = document.querySelector('#details-splitpanes')!
+  const panelWidth = panel.clientWidth
+  const iframeWidth = iframe.clientWidth
+  const iframePercent = Math.min((iframeWidth / panelWidth) * 100, 95)
+  const detailsPercent = 100 - iframePercent
+  detailSizes.value = [iframePercent, detailsPercent]
+}
+
+export function registerResizingListener(listener: ResizingListener) {
+  inject<(listener: ResizingListener) => void>(resizingSymbol)?.(listener)
```
Comment (sheremet-va) [inline packages/ui/client/composables/browser.ts:18]: isn't this just a `ResizingListener`?

File: packages/browser/src/client/orchestrator.ts [hunk 3]
```diff
@@ -44,11 +44,11 @@ function createIframe(container: HTMLDivElement, file: string) {
 
   iframe.style.display = 'block'
   iframe.style.border = 'none'
-  iframe.style.pointerEvents = 'none'
   iframe.setAttribute('allowfullscreen', 'true')
   iframe.setAttribute('allow', 'clipboard-write;')
 
   iframes.set(file, iframe)
+  container.className = 'scrolls'
```
Comment (sheremet-va) [inline packages/browser/src/client/orchestrator.ts:51]: should probably move that to line 190

**Review by sheremet-va — CHANGES_REQUESTED** [review]

File: packages/ui/client/composables/browser.ts (hunk 2 above)
Comment (userquin) [reply packages/ui/client/composables/browser.ts:18]: I'm goibg to configure it globally (like navigation conposable), thebuinis just a spa, dont need inject/provide

Comment (userquin) [reply packages/ui/client/composables/browser.ts:18]: Thisblistener is the callback called from the index.vue

Comment (sheremet-va) [reply packages/ui/client/composables/browser.ts:18]: I mean the type is repeated, you already defined it on the top

**Review by sheremet-va — APPROVED** [review]

**Pull request merged**